### PR TITLE
Merge updated dependencies

### PR DESCRIPTION
**Pull request by akramsabbah9**
Changelog:
- ported package.json at root of repository
- ported package.json in server directory
- updated ERESOLVE dependencies in client/package.json; added graphql

### CODE REVIEW

**Review by yulduzetta — APPROVED** [review]
LGTM! Will test the prod deploy once this is merged and the heroku+mongodb are set up.

**Review by benwade91 — APPROVED** [review]
looks good to me!

**Pull request merged**
